src/models/Project/createProject/index.js

```diff
@@ -35,6 +35,21 @@ const createProject = async ({
     errors.push({ field: 'name', message: nameValidation.error })
   }
 
+  const slug = slugify(trimmedName)
+  const existingProjectsWithSlug = await Project.query()
+    .where({ ownerId, slug })
+    .count()
+    .first()
+  const isDuplicateSlug = existingProjectsWithSlug.count > 0
+
+  if (isDuplicateSlug) {
+    errors.push({
+      field: 'name',
+      message:
+        'It seems like you’re alreading using this name for another project. Please use another one.'
```

Comment:
Could you add this constant to the `@constants/errors` module ?

Reply:
I think we should only put errors there when we re-use them elsewhere, what do you think?

Reply:
It doesn't really matter but in terms of wording it would centralize all error messages. I like to keep strings at one place (not in the code), but your point also makes sense, it doesn't really matter

Reply:
Yeah, I see why you’d want that. But then our errors file will get super big. :/